docs/platform/release-notes/tizen-5-5-m1.md

```diff
@@ -0,0 +1,630 @@
+- Bluetooth A2DP multipoint connection support
+  - Advanced Audio Distribution Profile (A2DP) sink target can support up to two different source devices.
+- Wi-Fi band steering
+  - Band steering mechanism between 2.4GHz and 5GHz of the same AP has been added.
+- API changes
+  - Intelligent network monitoring
+    - API to get network interface link information has been added.
+    - API to test network connection status (availability of gateway and DNS server) has been added.
+    - API to test reachability of HTTP server has been added.
+  - Wi-Fi manager
+    - API to get whether 5GHz band is supported has been added.
+    - API set to get Extensible Authentication Protocol (EAP) anonymous identity has been added.
+  - Bluetooth
+    - Privilege level of out of band (OOB) data getter API has been changed.
+  - Smart traffic control
+    - API to clone and destroy statistics info handle has been added.
+    - Legacy API to get statistics info asynchronously has been deprecated.
+    - API to get all statistics info has been added.
+    - Legacy API to get total statistics info has been deprecated.
+  - curl version up (7.62.0)
+  - C# API
+    - BluetoothLeDevice API set getting the remote device information has been added.
+    - Legacy C# Gatt API has been deprecated.
+    - New BluetoothGattClient C# API has been added.
+    - BluetoothLeDevice API set getting the remote device information has been deprecated.
+    - Bluetooth - Wrong AcceptStateChangedEventArgs class property and DestroyServerSocket API has been deprecated.
+    - Bluetooth - New SocketConnection class property has been added.
+    - Bluetooth - API to destroy Gatt server has been added.
+    - STC C# API has been added.
+    - API to check Wi-Fi scan state has been added.
+    - API to check cellular roaming state has been added.
+    - Wi-Fi - Exceptions to provide accurate error information has been added.
+
+
+## Security
+
+### New and changed features
+
+- Security-manager
+  - Smack rules will be loaded directly from security-manager DB. All rule files have been removed.
+- Privilege
+  - The following privileges have been added:
+    - autofillmanager, internal/buxton/systemsetting, filesystem.read, filesystem.write, windowsystem.admin, d2d.datasharing, d2d.remotelaunch
+- Privilege-info
+  - New API has been added.
+    - Getting several privilege information as list.
+
+
+## Service framework
+
+### New and changed features
+
+- Account-Manager
+  - Feature to query account information even after account deletion has been added.
+
+### Fixes
+
+- Account-Manager
+  - Unwanted D-Bus API calls have been removed.
+  - Memory leaks have been fixed.
+  - Fix to remove provider information from global account db has been added.
+- Sync-Manager
+  - Additional D-Bus error handling has been enabled.
+  - Potential defects have been fixed.
+- Calendar-service
+  - Support for calendar DB purge when Device is reset has been added.
```

Comment:
As per the comment given in PR 707, can we replace Device with device?

can we rephrase to:

Support for calendar DB purge when a device is reset has been added.

Reply:
will accept